tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/GradleProjectListener.java

```diff
@@ -0,0 +1,228 @@
+/**
```

Comment:
We should use the new ProjectConfigurator APIs that are in buildship instead of listening to the natureAdded event.  

See https://projects.eclipse.org/projects/tools.buildship/reviews/3.0.0-release-review